test/I.js

```diff
@@ -13,6 +13,7 @@ describe('I', function() {
 
   it('returns its argument', function() {
     eq(S.I([1, 2, 3]), [1, 2, 3]);
+    eq(S.I(['foo', 42]), ['foo', 42]);
```

Comment:
This works now because this patch adds [`$.Pair`][1] to Sanctuary's environment. #206 will fix the problem of "invalid" values in a general manner.


[1]: https://github.com/sanctuary-js/sanctuary-def#pair